releng/release-notes-next/addrepo-affects-bootstrap.feature

```diff
@@ -0,0 +1,8 @@
+The `--addrepo` option has been updated to affect both the bootstrap chroot
+installation and the buildroot installation, as requested in [issue#1414][].
+However, be cautious, as Mock [aggressively caches the bootstrap][issue#1289].
+Always remember to run `mock -r <chroot> --scrub=bootstrap` first.
+Additionally, as more chroots are being switched to `bootstrap_image_ready =
+True`, you'll likely need to use `--addrepo` **in combination with**
+`--no-bootstrap-image`; otherwise, the bootstrap chroot installation will remain
+unaffected.
```

Comment:
Should this go to man page too? Or at least abbrev version?

Reply:
Yes, but this is not a new thing - do you prefer to fix it here? Btw., #1454 should fix the problem what I'm trying to underline here.